src/20231106_AaveV3Ethereum_AaveFundingUpdates/AaveV3Ethereum_AaveFundingUpdates_20231106.sol

```diff
@@ -0,0 +1,133 @@
+import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
+import {AaveSwapper} from 'aave-helpers/swaps/AaveSwapper.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @title Aave Funding Updates
+ * @author efecarranza.eth
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x099f88e1728760952be26fcb8fc99b26c29336e6a109820b391751b108399ee5
+ * - Discussion: https://governance.aave.com/t/arfc-aave-funding-update/15194
+ */
+contract AaveV3Ethereum_AaveFundingUpdates_20231106 is IProposalGenericExecutor {
+  using SafeERC20 for IERC20;
+
+  AaveSwapper public constant SWAPPER = AaveSwapper(MiscEthereum.AAVE_SWAPPER);
+
+  uint256 public constant USDC_TO_SWAP = 200_000e6;
+  uint256 public constant USDC_TO_DEPOSIT = 1_500_000e6;
+  uint256 public constant USDT_TO_DEPOSIT = 750_000e6;
+  uint256 public constant DAI_TO_SWAP = 500_000e18;
+  uint256 public constant USDC_TO_MIGRATE = 900_000e6;
+
+  address public constant MILKMAN = 0x11C76AD590ABDFFCD980afEC9ad951B160F02797;
+  address public constant PRICE_CHECKER = 0xe80a1C615F75AFF7Ed8F08c9F21f9d00982D666c;
+
+  // https://etherscan.io/address/0x3f12643D3f6f874d39C2a4c9f2Cd6f2DbAC877FC
+  address public constant GHO_ORACLE = 0x3f12643D3f6f874d39C2a4c9f2Cd6f2DbAC877FC;
+
+  function execute() external {
+    AaveV2Ethereum.COLLECTOR.transfer(
+      AaveV2EthereumAssets.USDC_A_TOKEN,
+      address(this),
+      USDC_TO_MIGRATE
+    );
+
+    AaveV2Ethereum.POOL.withdraw(
+      AaveV2EthereumAssets.USDC_UNDERLYING,
+      USDC_TO_MIGRATE,
+      address(this)
+    );
+
+    IERC20(AaveV2EthereumAssets.USDC_UNDERLYING).forceApprove(
+      address(AaveV3Ethereum.POOL),
+      USDC_TO_MIGRATE
+    );
+
+    AaveV3Ethereum.POOL.deposit(
+      AaveV2EthereumAssets.USDC_UNDERLYING,
+      USDC_TO_MIGRATE,
+      address(AaveV3Ethereum.COLLECTOR),
+      0
+    );
+
+    AaveV3Ethereum.COLLECTOR.transfer(
+      AaveV3EthereumAssets.USDC_UNDERLYING,
+      address(SWAPPER),
+      USDC_TO_SWAP
+    );
+
+    AaveV3Ethereum.COLLECTOR.transfer(
+      AaveV3EthereumAssets.DAI_UNDERLYING,
+      address(SWAPPER),
+      DAI_TO_SWAP
+    );
+
+    // Deposit into V3
+
+    AaveV3Ethereum.COLLECTOR.transfer(
+      AaveV3EthereumAssets.USDC_UNDERLYING,
+      address(this),
+      USDC_TO_DEPOSIT
+    );
+    IERC20(AaveV3EthereumAssets.USDC_UNDERLYING).forceApprove(
+      address(AaveV3Ethereum.POOL),
+      USDC_TO_DEPOSIT
+    );
+    AaveV3Ethereum.POOL.deposit(
+      AaveV3EthereumAssets.USDC_UNDERLYING,
+      USDC_TO_DEPOSIT,
+      address(AaveV3Ethereum.COLLECTOR),
+      0
+    );
+
+    AaveV3Ethereum.COLLECTOR.transfer(
+      AaveV3EthereumAssets.USDT_UNDERLYING,
+      address(this),
+      USDT_TO_DEPOSIT
+    );
+    IERC20(AaveV3EthereumAssets.USDT_UNDERLYING).forceApprove(
+      address(AaveV3Ethereum.POOL),
+      USDT_TO_DEPOSIT
+    );
+    AaveV3Ethereum.POOL.deposit(
```

Comment:
we should deposit the USDT on Aave V2 right according to the latest comment on the forum?